package.json

```diff
@@ -55,9 +55,7 @@
     "type-check:watch": "yarn type-check -- --watch",
     "build": "yarn clean && rollup -c",
     "build:umd:min": "gzip index.min.js -c > index.min.js.gz",
-    "test": "yarn test:build && yarn test:run",
-    "test:build": "tsc --build tsconfig.test.json",
-    "test:run": "mocha -r tsconfig-paths/register --timeout 200000 output/tests/**/*.spec.js",
+    "test": "ts-mocha -p tsconfig.test.json --paths",
```

Comment:
We can proceed with ts-mocha as long as it allows us to debug the tests.

Reply:
That is something we should evaluate, need to know how you debug them so that we can test it :)

Reply:
Mainly via VSCode debugger. I do now that ts-mocha allows us to debug but currently at the cost of pointer precision (it seems the source maps are wrongly generated everytime) and some performance, but using it wouldn't be a problem if at least the source maps work fine. Also, will this impact the current CI? Current CI is already very slow and I'd not like to get it slower 🐌

Reply:
In short: debugging tests is already kinda broken, so VSCode debugging won't be a problem.